### DISCUSSION TITLE
Overlapping Clip recordings

### DISCUSSION THREAD

**Discussion by mannish-kumar** [blakeblackshear/frigate#3867] [Ask A Question]
I have recently upgraded my Frigate installation to 0.11 RC3 to resolve the issue (https://github.com/blakeblackshear/frigate/issues/2497). 

The freeze issue is resolved but I ended up with an old issue (https://github.com/blakeblackshear/frigate/issues/630) that is closed now.

I was using "max_seconds: 180" prior to upgrading to 0.11 RC3 and was not seeing any overlapping clips. I have removed this option as it is no more supported since 0.11 RC1.

Is there a way I can avoid overlapping recordings?

![image](https://user-images.githubusercontent.com/87553561/190984232-553a7ce0-be51-4726-aa1c-de36cfe047e3.png)

![image](https://user-images.githubusercontent.com/87553561/190984249-746f2de4-0e8e-4509-b4b0-be963a32a108.png)

**Comment by NickM-27** [comment]
Max seconds hasn't done anything for a long while, so even though you had it set it wasn't being used. 

The top person looks like it may be overlapping but for some reason frigate likely saw two different people. 

For the second one the person is facing one direction in one and a different direction in the other which seems like 2 separate events where they were out of the frame for some time.

**Reply by mannish-kumar** [reply]
Thanks Nick. Will increasing "post_capture: 5" say to 15 or 20 help? or any other way that can help me to avoid this.

Sorry If I am being too basic. I am trying to explore Frigate to replace my motioneye setup.

**Reply by NickM-27** [reply]
It really depends on what you're trying to solve / what you expect the behavior to be. 

You could increase `max_disappeared` but then you'd run the risk of two different people being a part of the same event.

**Reply by mannish-kumar** [reply]
I am trying to avoid duplicate clips getting recorded within a few seconds for the same event. Isn't  **max_disappeared** and **post_capture** similar? In both the cases the recording would stop after the seconds and frames specified.

Do you mean that if I use max_disappeared then only one person will be detected if two persons are part of same frame?

**Reply by NickM-27** [reply]
> I am trying to avoid duplicate clips getting recorded within a few seconds for the same event. Isn't max_disappeared and post_capture similar? In both the cases the recording would stop after the seconds and frames specified.

No they are not similar. `max_disappeared` is how long after the object is no longer detected before the event is considered over. 

`post_capture` just means once the event is over make sure there are at least that many seconds of recording that is apart of the event. It has nothing to do with the behavior of when an event starts or ends. 

> Do you mean that if I use max_disappeared then only one person will be detected if two persons are part of the same frame?

No. Let's use an example. Someone walks out their door and leaves the building, then 5 seconds later comes back inside. With the default max_disappeared this would likely be two events. With an increased max_disappeared this would have a better chance of appearing as a single event. 

The downside however is if one person walked out their door and left the building and another person came in a second later that would have a chance of being a part of 1 event instead of 2

**Comment by mannish-kumar** [comment]
Thank You Got it.

Appreciate your quick response.

I will monitor for some time and get back to you if any help is needed 🙂
